src/testScripts/sdkV2-scripts.ts

```diff
@@ -17,16 +17,14 @@ const ADMIN_ACCOUNTS = {
 const CONFIGS = {
   ethereum: {
     RPC_URL: process.env.ETH_RPC_URL,
-    cegaEntryAddress: '0x6891F6594Ec545077561Cc4Fa79a861db2cfBA9D' as types.EvmAddress,
-    cegaWrappingProxyAddress: '0xd940d163b1C29F8a8bFc7dd6300a94fF47AaDECB' as types.EvmAddress,
+    addressManager: '' as types.EvmAddress,
```

Comment:
use the address manager addresses here for ETH & ARB
https://www.notion.so/cega/QA-DCS-Deployments-a1d8fd0a7a304892a0b25875a9a79784

Reply:
added addresses from notion